src/algo.py

```diff
@@ -0,0 +1,34 @@
+# triggered by algo-is_prime snippet text
```

Comment:
do you want if `vscode_ext_gen.py` move to this folder too? People will be easier to find it

Reply:
I don't think so, because if someone checks our repo, `vscode_ext_gen.py` will catch there attention  and they will review this file, so i feel its better to place it in `root` directory rather than `src`

Reply:
Ok then